CODEOWNERS

```diff
@@ -463,6 +463,7 @@ build.json @home-assistant/supervisor
 /tests/components/http/ @home-assistant/core
 /homeassistant/components/huawei_lte/ @scop @fphammerle
 /tests/components/huawei_lte/ @scop @fphammerle
+/homeassistant/components/huawei_router/ @abmantis
```

Comment:
We might still want to consider not adding this file though, while it is something the should be fixed it might be out of scope for this PR

Reply:
The pre-commit hooks wouldn't let me push without uploading it. One for the admins to resolve.

Reply:
There's no integration in core called huawei_router. Do you have a custom integration installed in your dev environment?

Reply:
I'll try and clean my branch up. I've never done anything with that plugin, I'll see what I can do to get rid of it, maybe a full branch reset

Reply:
If I get it working I'll do a force-push and ditch this change altogether

Reply:
Got it working, this file is no longer in the PR. No idea how I pulled in those other files, but it's now clean.